src/WebEid.Security.Tests/Certificate/CertificateDataTest.cs

```diff
@@ -20,9 +18,9 @@ public void WhenOrganizationCertificateThenSubjectCNAndIdCodeAndCountryCodeExtra
         [Test]
         public void WhenOrganizationCertificateThenSubjectGivenNameAndSurnameExtractionFails()
         {
-            X509Certificate organizationCert = Certificates.GetOrganizationCert();
-            Assert.Throws<CertificateEncodingException>(() => organizationCert.GetSubjectGivenName());
-            Assert.Throws<CertificateEncodingException>(() => organizationCert.GetSubjectSurname());
+            var organizationCert = Certificates.GetOrganizationCert();
```

Comment:
Expicit type would be preferrable since it is not inherently clear from the method, what type of certificate is returned.